crates/biome_js_analyze/src/analyzers/nursery/use_node_import_protocol.rs

```diff
@@ -0,0 +1,87 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_js_syntax::JsModuleSource;
+use biome_rowan::AstNode;
+
+use crate::globals::node::NODE_BUILTINS;
+
+declare_rule! {
+    /// Enforces using the `node:` protocol for Node.js builtin modules.
+    ///
+    /// The rule marks traditional imports like import fs from `'fs'`; as invalid,
+    /// suggesting the format import fs from `'node:fs'`; instead.
+    ///
+    /// Source: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-node-protocol.md
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```js,expect_diagnostic
+    /// import fs from 'fs';
+    /// ```
+    ///
+    /// ```js,expect_diagnostic
+    /// import os from 'os';
+    /// ```
+    ///
+    /// ```js,expect_diagnostic
+    /// import path from 'path';
+    /// ```
+    ///
+    /// ### Valid
+    ///
+    /// ```js
+    /// import fs from 'node:fs';
+    ///
+    /// import os from 'node:os';
+    ///
+    /// import path from 'node:path';
+    /// ```
+    ///
+    pub(crate) UseNodeImportProtocol {
+        version: "next",
+        name: "useNodeImportProtocol",
+        recommended: false,
+    }
+}
+
+impl Rule for UseNodeImportProtocol {
+    type Query = Ast<JsModuleSource>;
+    type State = ();
+    type Signals = Option<Self::State>;
+    type Options = ();
+
+    fn run(ctx: &RuleContext<Self>) -> Self::Signals {
+        let binding: &JsModuleSource = ctx.query();
+        let Ok(module_name) = binding.inner_string_text() else {
+            return None;
+        };
```

Comment:
You can simply use `.ok()?` to obtain the result :)
```suggestion
        let module_name = binding.inner_string_text().ok()?;
```